tests/VectorException/host/host.c

```diff
@@ -162,7 +156,13 @@ int main(int argc, const char* argv[])
     OE_TEST(enc_test_cpuid_in_global_constructors(enclave) == OE_OK);
 
     test_vector_exception(enclave);
-    test_sigill_handling(enclave);
+
+    // SGX specific test is not available in simulation.
```

Comment:
Expand the comment to explain why test is SGX specific. Looks like it uses CPUID.

Reply:
Hi @anakrish I am not sure if this is the feature of SGX:
https://github.com/openenclave/openenclave/blob/f5952133bfe54bd97dfaa6af691ddfaea3c1f5a3/tests/VectorException/enc/sigill_handling.c#L73
while exception handling, in simulation mode rbx is not equal to tcs, but in SGX mode rbx is still tcs, although the above line succeed in both case. The different behaviors lead to different test result.